packages/shared/styles/components/organisms/SfStoreLocator.scss

```diff
@@ -12,39 +12,59 @@
     color: var(--c-dark-variant);
     @include font(
       --store-item-distance-font,
-      var(--font-light),
-      var(--font-base),
+      var(--font-weight--normal),
+      var(--font-size--base),
       1.4,
-      var(--font-family-primary)
+      var(--font-family--secondary)
     );
     span {
-      font-weight: var(--font-bold);
+      font-weight: var(--font-weight-bold);
       color: var(--c-text);
     }
   }
+  &__btn {
+    --button-background: var(--c-white);
```

Comment:
add to SfButton class --pure modifier and then you can remove most properties in btn and btn:active

Reply:
Thanks, great idea :)